app/lib/screens/identity_verification_screen.dart

```diff
@@ -1250,8 +1253,9 @@ class _IdentityVerificationScreenState
                 type: DialogType.Warning,
                 image: Icons.warning,
                 title: 'Not enough balance',
-                description:
-                    'Please fund your account with at least $minimumBalance TFTs.',
+                description: wallets.isEmpty
```

Comment:
```suggestion
                description: wallet.isEmpty
```

Reply:
combined wallets & wallet in one variable.